packages/lazy-import/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "@wordpress/lazy-import",
+	"version": "1.0.0-prerelease",
+	"description": "Lazily import a module, installing it automatically if missing.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/lazy-import/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/lazy-import"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "lib/index.js",
+	"types": "build-types",
+	"dependencies": {
+		"execa": "^4.0.2",
+		"npm-package-arg": "^8.0.1",
+		"semver": "^7.3.2"
```

Comment:
Similar note with `semver`:

https://github.com/WordPress/gutenberg/blob/65a4196b9df769e8529672f19e64c224af27095e/package.json#L176

It's clear now why npm install those modules in the package.

Reply:
> Similar note with `semver`:

Updated in c1f6a66053.